evm/src/main/java/org/hyperledger/besu/evm/operation/PushOperation.java

```diff
@@ -28,6 +30,8 @@
 
   private final int length;
 
+  private final OperationResult pushResponse;
```

Comment:
Curious, why pull this down from `successResponse` in parent? readability?

Reply:
Each push response has a different success, counting the PC to increment (2 for PUSH1, 33 for PUSH32).  The parent successResponse increments the PC by 1.